### ISSUE TITLE
_readpositionaxis read CRPIX from the fits header, but does not use it

### ISSUE THREAD

**Issue by gica3618** [richteague/gofish#13]
The method [_readpositionaxis](https://github.com/richteague/gofish/blob/f49334ead652b84e13cb9f195f81cc6600aa9ea3/gofish/gofish.py#L3255) reads the CRPIX value from the fits image header, but it does not use it. Instead, the origin of the axis is assigned to the middle pixel.
Not sure if it is a good idea to actually use the CRPIX value, but if it's not used, it shouldn't be read. Also, I couldn't find an explicit mention in the documentation that the center of the disk is assumed to equal the central pixel of the image (if x0 and y0 are set to 0).

**Comment by taniousm** [comment]
Yes, I join this comment. It would be convenient to get axis relative to the real center of the image, ie. CRPIX. For this, one can use this instead : `axis = 3600.0 * a_del * (np.arange(a_len) - (a_pix-1))`
on this line: https://github.com/richteague/gofish/blob/f49334ead652b84e13cb9f195f81cc6600aa9ea3/gofish/gofish.py#L3274
